docs/developer-docs/latest/developer-resources/cli/snippets/installation-prerequisites.md

```diff
@@ -1,5 +1,5 @@
 The installation requires the following software to be already installed on your computer:
 
-- [Node.js](https://nodejs.org): only LTS versions are supported (v14 and v16). Other versions of Node.js may not be compatible with the latest release of Strapi. The 16.x version is most recommended by Strapi.
+- [Node.js](https://nodejs.org): only LTS versions are supported (v14, v16 and v18). Other versions of Node.js may not be compatible with the latest release of Strapi. The 16.x version is most recommended by Strapi.
```

Comment:
```suggestion
- [Node.js](https://nodejs.org): only LTS versions are supported (v14, v16 and v18). Other versions of Node.js may not be compatible with the latest release of Strapi. Node v16.x version is recommended by Strapi.
```

Q: Is `Node v16.x` sill the recommended version? Can this be bumped to `node v18.x`?